File: Progamming Languages/C#/OOP/exercise_38/Children.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exercise_38
{
    public class Children:Person
    {
        
        public Children(){}

        public Children(string name, int age, string address, List<Book> listBook):base(name, age, address, listBook){

        }


        public override long moneyPay(){
            return ListBook.Count * 5000;
        }


        public void display(){
            Console.WriteLine("Children!");
            base.display();
        }


    }
}

File: Progamming Languages/C++/OOP Person/Children.cpp
#include<iostream>
#include "Person.cpp"
using namespace std;
class Children : public Person{
    public:
    Children(){}
    Children(string name, int age, string add, vector<Book> vb):Person(name,age, add, vb){}
    long moneyPay(){
        return Person::getVB().size()*5000;
    }
    void display(){
        cout<<"Children!"<<endl;
        Person::display();
    }
};